README.md

```diff
@@ -372,6 +372,41 @@ monday.execute('triggerFilesUpload', {
 })
 ```
 
+#### Open modal
+Opens a new modal window as an iFrame.
+
+**type**
+`'openAppFeatureModal'`
+
+**params**
+
+| Parameter|Type | Description | Required | Default Value |
+| --- | --- | --- | --- | --- |
+| url | String | The URL of the page displayed in the modal | No | current iFrame's URL |
+| urlPath | String | Subdirectory or path of the URL to open | No | |
+| urlParams | Object | Query parameters for the URL | No | |
+| width | String | The width of the modal | No | "0px" |
+| height | String | The height of the modal | No | "0px" |
+
+**Example**
+```javascript
+monday.execute('openAppFeatureModal', { url: "https://example.com", urlParams: { name: "Dipro" }, urlPath: "/monday", width: "800px", height: "600px" })
```

Comment:
@helenhl20 

There is a callback that you can use from this method. It will tell you if someone closed the modal from inside the modal so you can run some logic in the app window

```suggestion
monday.execute('openAppFeatureModal', { url: "https://example.com", urlParams: { name: "Dipro" }, urlPath: "/monday", width: "800px", height: "600px" })monday.execute('openAppFeatureModal', { url, urlPath, urlParams, height, width }).then((res) => {
console.log(res.data);
// {"close": true}
});;
```

Reply:
I would even add a separate note, that urlPath needs to be a relevant URL and not absolute.

Do we enforce it somehow, that the developer will not pass the absolute URL?